source/issues-and-help.txt

```diff
@@ -0,0 +1,50 @@
+=============
+Issues & Help
+=============
+
+.. default-domain:: mongodb
+
+We are lucky to have a vibrant MongoDB C#/.NET community
+that includes users with varying levels of experience using
+the driver. We find the quickest way to get support for general
+questions is through the `MongoDB Community Forums
```

Comment:
Suggestion:

In the original version, "We are lucky to have" refers to the community that specifically has varying levels of experience, but it's not clear why the users with varying levels of experience makes MongoDB lucky or how it qualifies the users of the forums as the quickest way to get support. Instead, consider omitting or explaining the connection.
 
I see this is essentially what is written on the Manual "Technical Support" page, but that doesn't mean it's clear!

```suggestion
of users. We recommend seeking support for general
questions through the `MongoDB Community Forums
```